.github/workflows/pr-review.yml

```diff
@@ -0,0 +1,43 @@
+name: PR Preview
+on:
+  pull_request:
+    types: [synchronize]
```

Comment:
I find "synchronize" to be a misnomer. This runs on each update (commit) to the source branch.

Reply:
https://github.community/t/what-is-a-pull-request-synchronize-event/14784